src/setup/rgw_start.py

```diff
@@ -27,8 +27,13 @@ class RgwStart:
     def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
-        except OSError as e:
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
```

Comment:
Check if we can directly use this command as entrypoint, if nothing significant being achieved in rgw entrypoint script/s
@sachitanands @rahul-tripathi-git

Reply:
we need to do pre-processing activity for fetching the config file path & log file path. and for this need confstore to fetch those values...hence we need this wrapper.